cli_tools/common/utils/storage/scratch_bucket_creator.go

```diff
@@ -124,3 +137,14 @@ func (c *ScratchBucketCreator) formatScratchBucketName(project string, location
 	}
 	return strings.ToLower(bucket)
 }
+
+func getRegion(zone string) (string, error) {
```

Comment:
This is a copy paste from param_utils. Reuse that method.

Reply:
param_utils imports storage_utils, so I need to avoid circled dependency.
As a exposed function, param_utils is the correct place to put "GetRegion". But we need to use it in storage_utils.
It's not worth adding an extra common lib for that.

Reply:
I think adding an extra common lib to avoid code duplication of any decent size (and this one is decent) is worth it.

Reply:
Sounds good. Let me move it.

Reply:
Sounds good, resetting the value